roles/ora-host/tasks/main.yml

```diff
@@ -80,6 +80,13 @@
   when:
     - install_os_packages|bool and ansible_distribution == 'OracleLinux'
 
+- name: Stat OL8 repo
+  stat:
+    path: /etc/yum.repos.d/oracle-linux-ol8.repo
+  register: oraclelinux_repo_oel8
+  when:
+    - install_os_packages|bool and ansible_distribution == 'OracleLinux'
```

Comment:
Do we expect RHEL8 to be in a future PR?

Reply:
That's correct. Once we get all the changes approved via individual PRs (keeping the PRs small for an individual issue), I plan to reimage the syd2 host to RHEL 8.5 and attempt the same RAC install.